setup/nextcloud.sh

```diff
@@ -333,6 +354,11 @@ sudo -u www-data \
 	php$PHP_VER /usr/local/lib/owncloud/occ app:disable photos dashboard activity \
 	| (grep -v "No such app enabled" || /bin/true)
 
+# Now set the config to read-only.
+# Do this only at the very bottom when no further occ commands are needed.
+sed -i'' "s/'config_is_read_only'\s*=>\s*false/'config_is_read_only' => true/" $STORAGE_ROOT/owncloud/config.php
+echo "Read-onlyness restored."
```

Comment:
End users likely won't know what this means and will likely just cause more questions on the forums/github

Reply:
Removed.